src/common/components/Input.jsx

```diff
@@ -35,12 +45,15 @@ export default class Input extends Component {
 }
 
 Input.propTypes = {
+    id: PropTypes.string,
     isValid: PropTypes.bool,
     name: PropTypes.string,
     onChange: PropTypes.func,
     placeholder: PropTypes.string,
     type: PropTypes.string.isRequired,
+    validationFunc: PropTypes.func,
     value: PropTypes.string,
+    disabled: PropTypes.bool,
```

Comment:
This change must be in the next commit: https://github.com/tolstoyevsky/cusdeb-web-client/pull/16/commits/a48bf13e3b229b89654a74b7a7ba3b6d09d39b45